src/modules/new-request-form/translations/en-us.yml

```
@@ -61,6 +61,7 @@ parts:
       title: "[A11Y] Hidden label for screen readers, describing an empty option in a dropdown"
       screenshot: "https://drive.google.com/file/d/1enaOJceIEIVn_uYqvaF4AHlE1_RIpjqe/view?usp=drive_link"
       value: "Select an option"
+      obsolete: "2025-19-03"
```

```suggestion
      obsolete: "2025-03-19"
```
https://zendeskdev.zendesk.com/hc/en-us/articles/7381249461786-How-do-I-make-a-translation-string-obsolete-or-delete-it-in-internationalization-i18n :)

---

hah thanks, what was I thinking 😄